frontend/test/metabase/scenarios/dashboard/reproductions/21597-query-build-card-save-modal.cy.spec.js

```diff
@@ -29,7 +29,7 @@ describe("display the relevant error message in save question modal (metabase#21
 
     // Second DB (copy)
     cy.visit("/");
-    cy.get(".Nav .Icon-gear").click();
+    cy.icon("gear").click();
```

Comment:
One of the E2E tests were failing because of the gear icon position change.
https://github.com/metabase/metabase/runs/7079411239?check_suite_focus=true